examples/co2/co2.py

```diff
@@ -0,0 +1,181 @@
+"""
+Add a SCD41 sensor breakout to PicoVision to make a handy CO2 detector!
+https://shop.pimoroni.com/products/scd41-co2-sensor-breakout
+Press A to reset the high/low values.
+
+You will need the OpenSans-Bold.af font saved to your PicoVision alongside this script.
+"""
+
+from picovision import PicoVision, PEN_RGB555
+from pimoroni import PICOVISION_I2C_PINS
+from pimoroni_i2c import PimoroniI2C
+import breakout_scd41
+from picovector import Polygon, PicoVector, ANTIALIAS_X16
+
+i2c = PimoroniI2C(**PICOVISION_I2C_PINS)
```

Comment:
Interesting - does this work OK?  I expected you would have to do `display.get_i2c()` to avoid breaking stuff.

Reply:
It seems to - didn't realise there was a special function for PicoVision...